scripts/convert_metadata.py

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env python
+
 # Python library imports
 import sys
 import os.path
 import re
 import logging
 from collections import OrderedDict
+import logging
```

Comment:
Why is this line necessary?

Reply:
Uups, I had introduced the logger earlier on and then merged in your recent commit that did the same.